examples/example-app-v12/src/app/hero/hero-detail.component.ts

```diff
@@ -0,0 +1,51 @@
+/* tslint:disable:member-ordering */
```

Comment:
this comment should be removed. Would you please also correct any other places where tslint comments appear?